mscore/inspector/inspector.cpp

```diff
@@ -160,13 +160,10 @@ void Inspector::setElements(const QList<Element*>& l)
 
             bool sameTypes = true;
             for (Element* ee : _el) {
-                  if (_element->type() != ee->type())
+                  if ((_element->type() != ee->type()) && // different and
+                      (!_element->isSlurTieSegment() && !ee->isSlurTieSegment()) || // neither Slur nor Tie either side, or
+                      (ee->isNote() && toNote(ee)->chord()->isGrace() != toNote(_element)->chord()->isGrace())) // HACK
```

Comment:
Hmm something indeed isn't right there, needs to be `(!_element->isSlurTieSegment() || !ee->isSlurTieSegment())`